packages/babel-core/src/config/validation/plugins.ts

```diff
@@ -11,6 +11,10 @@ import type {
   OptionPath,
   RootPath,
 } from "./option-assertions";
+import type { ParserOptions } from "@babel/parser";
+import type { Visitor } from "@babel/traverse";
+import PluginPass from "../../transformation/plugin-pass";
```

Comment:
We are only using it as a type
```suggestion
import type PluginPass from "../../transformation/plugin-pass";
```